daemon/cmd/daemon_main.go

```diff
@@ -1580,6 +1580,10 @@ func initKubeProxyReplacementOptions() {
 
 	if option.Config.EnableNodePort &&
 		option.Config.NodePortAcceleration != option.NodePortAccelerationNone {
+		if option.Config.Tunnel != option.TunnelDisabled {
+			log.Fatal("Cannot use NodePort acceleration with tunneling. Only direct routing mode supported at this point.")
```

Comment:
To make it clear for users how they can fix this we should probably end this error message with `supported at this point. Either run cilium-agent with '--%s=%s' or '--%s=%s'", option.NodePortRange, option.NodePortAccelerationNone, option.TunnelName, option.TunnelDisabled)`